terraform/modules/s3_migration/kms.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_kms_key" "this" {
+  description         = "KMS key used for the Secrets Manager secrets."
+  enable_key_rotation = true
+}
+
+resource "aws_kms_alias" "this" {
+  name          = "alias/${var.prefix}-db-migration-secretsmanager-secrets"
```

Comment:
Should be `...s3-migration` rather than `...db-migration`

Reply:
Ooh, good point I missed that one.